x-pack/test/functional/services/cases/common.ts

```diff
@@ -29,125 +28,11 @@ export function CasesCommonServiceProvider({ getService, getPageObject }: FtrPro
       });
     },
 
-    /**
-     * it creates a new case from the create case page
-     * and leaves the navigation in the case view page
-     *
-     * Doesn't do navigation. Only works if you are already inside a cases app page.
-     * Does not work with the cases flyout.
-     */
-    async createCaseFromCreateCasePage({
-      title = 'test-' + uuid.v4(),
-      description = 'desc' + uuid.v4(),
-      tag = 'tagme',
-    }: {
-      title: string;
-      description: string;
-      tag: string;
-    }) {
-      await this.openCreateCasePage();
-
-      // case name
-      await testSubjects.setValue('input', title);
-
-      // case tag
-      await comboBox.setCustom('comboBoxInput', tag);
-
-      // case description
-      const descriptionArea = await find.byCssSelector('textarea.euiMarkdownEditorTextArea');
-      await descriptionArea.focus();
-      await descriptionArea.type(description);
-
-      // save
-      await testSubjects.click('create-case-submit');
-
-      await testSubjects.existOrFail('case-view-title');
-    },
-
-    /**
-     * Goes to the first case listed on the table.
-     *
-     * This will fail if the table doesn't have any case
-     */
-    async goToFirstListedCase() {
-      await testSubjects.existOrFail('cases-table');
-      await testSubjects.click('case-details-link');
-      await testSubjects.existOrFail('case-view-title');
-    },
-
-    /**
-     * Marks a case in progress via the status dropdown
-     */
-    async markCaseInProgressViaDropdown() {
-      await this.openCaseSetStatusDropdown();
-
-      await testSubjects.click('case-view-status-dropdown-in-progress');
-
-      // wait for backend response
-      await testSubjects.existOrFail('header-page-supplements > status-badge-in-progress', {
-        timeout: 5000,
-      });
-    },
-
-    /**
-     * Marks a case closed via the status dropdown
-     */
-    async markCaseClosedViaDropdown() {
-      this.openCaseSetStatusDropdown();
-
-      await testSubjects.click('case-view-status-dropdown-closed');
-
-      // wait for backend response
-      await testSubjects.existOrFail('header-page-supplements > status-badge-closed', {
-        timeout: 5000,
-      });
-    },
-
-    /**
-     * Marks a case open via the status dropdown
-     */
-    async markCaseOpenViaDropdown() {
+    async changeCaseStatusAndVerify(status: CaseStatuses) {
```

Comment:
There are many ways to change a case status so this function is short on clarity. perhaps `changeCaseStatusViaDropdownAndVerify` ? considering there are at least other 2 places in the UI and the API you can change a case status.